public/electron.js

```diff
@@ -143,4 +160,6 @@ app.on('activate', () => {
   }
 })
 
-app.on('quit', () => queues.close())
+app.on('quit', () => {
+  queues.close()
+})
```

Comment:
How come this was changed?

Reply:
Because I did some test inside this block of codes. Reverted it back now.